cmd/flipt/evaluate.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"github.com/spf13/cobra"
+	"go.flipt.io/flipt/rpc/flipt"
+	"go.flipt.io/flipt/rpc/flipt/evaluation"
+	sdk "go.flipt.io/flipt/sdk/go"
+	"go.uber.org/zap"
+)
+
+type evaluateCommand struct {
+	address       string
+	requestID     string
+	entityID      string
+	namespace     string
+	watch         bool
+	token         string
+	interval      time.Duration
+	contextValues []string
+}
+
+func newEvaluateCommand() *cobra.Command {
+	c := &evaluateCommand{}
+
+	cmd := &cobra.Command{
+		Use:   "evaluate [flag]",
+		Short: "Evaluate the flag",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.run,
+	}
+
+	cmd.Flags().StringVarP(
+		&c.namespace,
+		"namespace", "n",
+		"default",
+		"flag namespace.",
+	)
+	cmd.Flags().StringVarP(
+		&c.entityID,
+		"entity-id", "e",
+		uuid.Must(uuid.NewV4()).String(),
+		"evaluation request entity id.",
+	)
+	cmd.Flags().StringVarP(
+		&c.requestID,
+		"request-id", "r",
+		"",
+		"evaluation request id.",
+	)
+
+	cmd.Flags().StringArrayVarP(
+		&c.contextValues,
+		"context", "c",
+		[]string{},
+		"evaluation request context as key=value.",
+	)
+
+	cmd.Flags().StringVarP(
+		&c.address,
+		"address", "a",
+		"http://localhost:8080",
+		"address of Flipt instance to evaluate.",
```

Comment:
```suggestion
		"address of Flipt instance.",
```

i think we might need to update some other places where we use `address` and `token` descriptions that could be a bit shorter. i can do that in another PR